### DISCUSSION TITLE
delete page problem

### DISCUSSION THREAD

**Discussion by qunshen** [oqtane/oqtane.framework#1559] [General] [answered]
After delete a page, if you add another one with the same name as the one deleted, the system does not allow you to do so, saying a page with the same already exists.

I checked db, find that the page deletd just marked as deleted, not deleted from database!

I would like to suggest that page should be deleted from database, or provide a UI to restore the page deleted

Thanks

**Comment by leigh-pointer** [comment, marked as answer]
@qunshen this is correct.  After deleting Pages or Modules from the system they are marked for deletion.  To physically delete the Page or Module you must empty the recycle bin which is found in the Admin Dashboard.

![image](https://user-images.githubusercontent.com/9447612/125897264-ba732126-3c6a-4a41-b3dc-b4e5a51db200.png)

![image](https://user-images.githubusercontent.com/9447612/125897396-1c9768bd-1dda-41d9-8dad-a38a716cc6ca.png)

**Reply by qunshen** [reply]
got it! thank you

**Reply by leigh-pointer** [reply]
@qunshen glad to help, can you mark this as answered please?
